gcb/release-krel/cloudbuild.yaml

```diff
@@ -0,0 +1,70 @@
+timeout: 14400s
+
+#### SECURITY NOTICE ####
+# Google Cloud Build (GCB) supports the usage of secrets for build requests.
+# Secrets appear within GCB configs as base64-encoded strings.
+# These secrets are GCP Cloud KMS-encrypted and cannot be decrypted by any human or system
+# outside of GCP Cloud KMS for the GCP project this encrypted resource was created for.
+# Seeing the base64-encoded encrypted blob here is not a security event for the project.
+#
+# More details on using encrypted resources on Google Cloud Build can be found here:
+# https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-secrets-credentials
+#
+# (Please do not remove this security notice.)
+secrets:
+- kmsKeyName: projects/k8s-releng-prod/locations/global/keyRings/release/cryptoKeys/encrypt-0
+  secretEnv:
+    GITHUB_TOKEN: CiQAIkWjMMb2OjCHCt1hHer4ZBqqD1Mj7+dfTpvkkuzS64nrGm0SUQBLz1D+4Zwp6te6JByO/zvGpHMhW6/i3BJMRYXtNTkyBGMnZuH+J3gQfhcGL2HXjr75lHaCxF4bxPr45xen5D/Kk1+paL+XpOU3KJIADy7uGQ==
+
+steps:
+- name: gcr.io/cloud-builders/git
+  dir: "go/src/k8s.io"
+  args:
+  - "clone"
+  - "--branch=${_TOOL_BRANCH}"
+  - "https://github.com/${_TOOL_ORG}/${_TOOL_REPO}"
+
+- name: k8s.gcr.io/build-image/kube-cross:${_KUBE_CROSS_VERSION}
+  dir: "go/src/k8s.io/release"
+  env:
+  - "GOPATH=/workspace/go"
+  - "GOBIN=/workspace/bin"
+  args:
+  - "./compile-release-tools"
+
+- name: gcr.io/k8s-staging-releng/k8s-cloud-builder:${_KUBE_CROSS_VERSION}
+  dir: "/workspace"
+  env:
+  - "TOOL_ORG=${_TOOL_ORG}"
+  - "TOOL_REPO=${_TOOL_REPO}"
+  - "TOOL_BRANCH=${_TOOL_BRANCH}"
+  secretEnv:
+  - GITHUB_TOKEN
+  args:
+  - "bin/krel"
+  - "release"
+  - "${_NOMOCK}"
+  - "--type=${_TYPE}"
+  - "--branch=${_RELEASE_BRANCH}"
+  - "--build-version=${_BUILDVERSION}"
+
+tags:
+- ${_GCP_USER_TAG}
+- ${_RELEASE_BRANCH}
+- ${_BUILD_POINT}
+- ${_NOMOCK_TAG}
+- RELEASE
+- ${_GIT_TAG}
+- ${_TYPE_TAG}
+- ${_MAJOR_VERSION_TAG}
+- ${_MINOR_VERSION_TAG}
+- ${_PATCH_VERSION_TAG}
+- ${_KUBERNETES_VERSION_TAG}
```

Comment:
Let's add a tag here for the new implementation; that way we can filter the GCB runs when we need to get stats.

Same below for the staging GCB config.

Reply:
Yep, added a `NO_ANAGO` tag: https://console.cloud.google.com/cloud-build/builds/dcb42fff-905b-4f53-83d1-d3135a4af57b?project=648026197307